### PR TITLE
make test github.com/padraic/mockery/pull/456/ pass under hhvm

### PR DESCRIPTION
**Pull request by zodiac**
- [ ] CR by @mflores13 cc @karoun

### CODE REVIEW

File: library/Mockery/Container.php [hunk 1]
```diff
@@ -195,6 +195,11 @@ public function mock()
 
         $builder->addBlackListedMethods($blocks);
 
+        if ($class === 'Exception' || is_subclass_of($class, 'Exception')) {
```
Comment (karoun) [inline library/Mockery/Container.php]: Can we add a comment about why this is necessary?

Comment (zodiac) [inline library/Mockery/Container.php]: The if statement or the addBlackListMethod call?

Comment (karoun) [inline library/Mockery/Container.php]: Ideally both!

**Comment by zodiac** [comment]
Closed by https://github.com/padraic/mockery/commit/1354c034299f3fcb69c95e4f92933803639c922f

**Pull request closed without merging**